content/integrations/azure.md

```diff
@@ -18,25 +18,73 @@ sidebar:
 
 ### Overview
 
-Datadog's Azure integration currently monitors Virtual Machines (diagnostics must be enabled), and SQL Databases. For any other service, please reach out to [support@datadoghq.com][3].
+Datadog's Azure integration currently monitors Virtual Machines, SQL Databases, and App Services. For any other service, please reach out to [support@datadoghq.com][3].
 Datadog's agent can also be deployed on your Azure virtual machines.
 
 
 ### Configure Azure Monitoring
 {: #monitoring}
 
+Integrating Datadog with Microsoft Azure is a two step process. The first step is creating a web application in your Active Directory and passing the correct credentials to Datadog, and the second step is giving this application read-access to any subscriptions you would like to monitor.
+
+#### Getting your tenant ID
+1. Go to [manage.windowsazure.com][5]
+2. Once the URL redirects, copy the tenant name from the URL. It is the text in between the @ and # symbol <img src="/static/images/azure/azure_tenant_url.png" style="width: 75%;display:block;"/>
+3. Paste the Tenant Name in the form on the Datadog setup page <img src="/static/images/azure/tenant_name_form.png" style="width: 50%;display:block;display:block;"/>
+
+#### Creating the Web Application
+
+1. Navigate to the “Active Directory” tab on the left-hand menu <img src="/static/images/azure/active_directory_icon.png" style="width: 25%;display:block;"/>
+2. Click on the Active Directory you want to monitor <img src="/static/images/azure/active_directory_name_click.png" style="width: 50%;display:block;"/>
+3. Click on "Applications" <img src="/static/images/azure/application_click.png" style="width: 50%;display:block;"/>
```

Comment:
@technovangelist @jyee Do we prefer [markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images) for including images over html?